sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/SdkMeterProviderTest.java

```diff
@@ -0,0 +1,460 @@
+                    Collections.singletonList(
+                        MetricData.LongPoint.create(
+                            testClock.now(), testClock.now(), Labels.empty(), 1)))),
+            MetricData.createLongSum(
+                RESOURCE,
+                INSTRUMENTATION_LIBRARY_INFO,
+                "testDoubleCounter",
+                "",
+                "1",
+                MetricData.LongSumData.create(
+                    /* isMonotonic= */ true,
+                    MetricData.AggregationTemporality.CUMULATIVE,
+                    Collections.singletonList(
+                        MetricData.LongPoint.create(
+                            testClock.now(), testClock.now(), Labels.empty(), 1)))),
+            MetricData.createLongSum(
+                RESOURCE,
+                INSTRUMENTATION_LIBRARY_INFO,
+                "testLongUpDownCounter",
+                "",
+                "1",
+                MetricData.LongSumData.create(
+                    /* isMonotonic= */ true,
+                    MetricData.AggregationTemporality.CUMULATIVE,
+                    Collections.singletonList(
+                        MetricData.LongPoint.create(
+                            testClock.now(), testClock.now(), Labels.empty(), 1)))),
+            MetricData.createLongSum(
+                RESOURCE,
+                INSTRUMENTATION_LIBRARY_INFO,
+                "testDoubleUpDownCounter",
+                "",
+                "1",
+                MetricData.LongSumData.create(
+                    /* isMonotonic= */ true,
+                    MetricData.AggregationTemporality.CUMULATIVE,
+                    Collections.singletonList(
+                        MetricData.LongPoint.create(
+                            testClock.now(), testClock.now(), Labels.empty(), 1)))),
+            MetricData.createLongSum(
+                RESOURCE,
+                INSTRUMENTATION_LIBRARY_INFO,
+                "testLongValueRecorder",
+                "",
+                "1",
+                MetricData.LongSumData.create(
+                    /* isMonotonic= */ true,
+                    MetricData.AggregationTemporality.CUMULATIVE,
+                    Collections.singletonList(
+                        MetricData.LongPoint.create(
+                            testClock.now(), testClock.now(), Labels.empty(), 1)))),
+            MetricData.createLongSum(
+                RESOURCE,
+                INSTRUMENTATION_LIBRARY_INFO,
+                "testDoubleValueRecorder",
+                "",
+                "1",
+                MetricData.LongSumData.create(
+                    /* isMonotonic= */ true,
+                    MetricData.AggregationTemporality.CUMULATIVE,
+                    Collections.singletonList(
+                        MetricData.LongPoint.create(
+                            testClock.now(), testClock.now(), Labels.empty(), 1)))));
+  }
+
+  @Test
+  void collectAllASyncInstruments() {
```

Comment:
nit : collectAllAsyncInstruments  (lower-case the 's' in async)

Reply:
Done.